src/proto/device_path.rs

```diff
@@ -0,0 +1,84 @@
+//! `DevicePath` protocol
+
+use crate::{proto::Protocol, unsafe_guid};
+
+/// DevicePath protocol. This can be opened on a `LoadedImage.device()` handle
```

Comment:
Let's say `Device path protocol`, since here we're referring to the protocol's name, not it's identifier.